h2o-py/conda/h2o-client/meta.yaml

```diff
@@ -22,5 +22,5 @@ requirements:
 
 about:
   home: https://github.com/h2oai/h2o-3.git
-  license: Apache License Version 2.0
-  license_family: Apache
+  license: GNU Affero General Public License v3
+  license_family: AGPL
```

Comment:
Should this be v3.0?

Reply:
I don't think so
https://github.com/bioconda/bioconda-recipes/issues/1581

Reply:
I think conda have the same restrictions as pypi